src/Components/Clock/Clock.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+
+const Clock: React.FC = () => {
+
+    const [count, setCount] = React.useState<number>(0);
+    const [isRunning, setIsRunning] = React.useState<boolean>(false);
+
+    React.useEffect(() => {
+        if (isRunning) {
+            const interval = setInterval(() => {
+                setCount(prevCount => prevCount + 1);
+            }, 1000);
+            return () => clearInterval(interval);
+        }
+    }, [isRunning]);
+
+    const start = () => setIsRunning(true);
+
+    const stop = () => setIsRunning(false);
+
+    const reset = () => setCount(0);
+
+    return (
+        <div>
+            <div>
+                {Math.floor(count / 3600)} : {Math.floor(count / 60) % 60} : {count % 60}
```

Comment:
Have you tested with this value?
`2hrs 5mins 0secs`
Can you see `02:05:00` on your screen?